python/tests/expressions/test_visitors.py

```diff
@@ -954,36 +919,33 @@ def test_manifest_evaluator_less_than_or_equal_overlap():
             upper_bound=_to_byte_buffer(StringType(), "b"),
         )
     )
+    assert _ManifestEvalVisitor(string_schema, expr).eval(manifest)
 
-    assert _create_manifest_evaluator(expr).eval(manifest)
-
-
-def test_manifest_evaluator_less_than_or_equal_all_null():
-    expr = BoundLessThanOrEqual(
-        term=BoundReference(
-            field=NestedField(field_id=1, name="foo", field_type=StringType(), required=False),
-            accessor=Accessor(position=0, inner=None),
-        ),
-        literal=StringLiteral("a"),
-    )
 
+def test_manifest_evaluator_less_than_or_equal_all_null(string_schema: Schema):
+    expr = LessThanOrEqual[str](Reference("col_str"), StringLiteral("a"))
     manifest = _to_manifest_file(
-        PartitionFieldSummary(contains_null=False, contains_nan=False, lower_bound=None, upper_bound=None)
+        PartitionFieldSummary(contains_null=True, contains_nan=False, lower_bound=None, upper_bound=None)
     )
-
     # All null
-    assert not _create_manifest_evaluator(expr).eval(manifest)
+    assert not _ManifestEvalVisitor(string_schema, expr).eval(manifest)
 
 
-def test_manifest_evaluator_less_than_or_equal_no_match():
-    expr = BoundLessThanOrEqual(
-        term=BoundReference(
-            field=NestedField(field_id=1, name="foo", field_type=StringType(), required=False),
-            accessor=Accessor(position=0, inner=None),
-        ),
-        literal=StringLiteral("a"),
+def test_manifest_evaluator_less_than_or_equal_match(string_schema: Schema):
```

Comment:
This looks like the same test as `test_manifest_evaluator_less_than_or_equal_on_bound`.